docs/source/conf.py

```diff
@@ -12,6 +11,11 @@
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
 
+import sphinx.ext.doctest
+from sphinx import addnodes
+from sphinx.util.docfields import TypedField
+from docutils import nodes
+import rising
```

Comment:
```suggestion
```

This needs to stay below, since this will only be possible after `sys.path.insert(0, os.path.abspath(PATH_ROOT))` 
@Borda , how does one prevent this line from being formatted?